v20.1/cockroachcloud-cluster-management.md

```diff
@@ -32,11 +28,26 @@ For each node, the page displays the node's `Name`, nested under its region.
 
 From the **Cluster Overview** page, you can connect to your cluster. For more information, see [Connect to Your CockroachCloud Cluster](cockroachcloud-connect-to-your-cluster.html).
 
-## Manage cluster configuration
+## Add or remove nodes from a cluster
+
+You can add or remove nodes from your cluster through the Console.
+
+{{site.data.alerts.callout_info}}
+At this time, you cannot use the Console to scale up a single-node cluster or scale down to a single-node cluster. For single-node cluster changes, contact [Support](https://support.cockroachlabs.com).
```

Comment:
I might change the second sentence to say "for these types of changes", since you've already described them in the first sentence.  Also, "single-node cluster changes" reads as ambiguous to me - it could refer to other changes to single-node clusters other than those described in the previous sentence.

Reply:
Yeah..I struggled with this phrasing. Thanks for the suggestion! Done!